scripts/delete_openshift_project.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+# Deletes Openshift project stuck in the finalize state
+# Usage: delete_openshift_project enmasse-infra ec2-x-x-x-x.zone.compute.amazonaws.com:8443
+
+TOKEN=$(oc whoami -t)
+# This doesn't work with ec2-x-x-x-x.zone.compute.amazonaws.com format of host names
+#ENDPOINT=$(oc config current-context | cut -d/ -f2 | tr - .)
+ENDPOINT="$2"
+PROJECT="$1"
+
+kubectl get namespace "$PROJECT" -o json | jq 'del(.spec.finalizers[] | select("kubernetes"))' | curl --insecure -k -H "Content-Type: application/json" -H "Authorization: Bearer $TOKEN" -X PUT --data-binary @- https://$ENDPOINT/api/v1/namespaces/$PROJECT/finalize
```

Comment:
on server where is openshift installed this will fail because kubectl is not installed

Reply:
`kubectl` is available in the same dir as `oc` in the download archive, so it should work.I'm not sure if it work with minishift though. Thought it'd be useful as a util script, if not I can put it somewhere else.

Reply:
don't understand which download archive you mean?

Reply:
I meant on Openshift client tools download